src/containers/Bounty/SubmissionsAndCommentsCard.module.scss

```diff
@@ -85,3 +85,22 @@
   text-align: center;
   padding-top: $l-spacing;
 }
+
+.declinedNoteText {
+  background-color: $brand-nearWhite;
+  position: relative;
+  margin-right: -$l-spacing;
+  margin-left: -$l-spacing;
+  padding: $base-spacing;
+}
+
+.bottomBorder {
+  border-bottom: 0 solid $brand-nearWhite;
```

Comment:
I'm not really sure but once I increase it, it no longer matches the other border for girdItems component. But it is visible even though it's 0. Maybe because of solid?